frontend/src/marketplace/EntryList.tsx

```diff
@@ -22,49 +23,69 @@ export default function EntryList({
 }: EntryListProps) {
   const theme = useTheme()
 
-  const entriesDisplay = useMemo(() => {
-    return entries.map((entry, index) => (
-      <Fragment key={entry.id}>
-        <Stack direction='row'>
-          <Link
-            sx={{ textDecoration: 'none', whiteSpace: 'nowrap', textOverflow: 'ellipsis', overflow: 'hidden' }}
-            href={`${entry.kind}/${entry.id}`}
-          >
-            <Typography
-              variant='h5'
-              sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.primary.main }}
+  const Row = memo(({ data, index, style }: { data: EntrySearchResult[]; index: number; style: CSSProperties }) => {
+    const entry = data[index]
+
+    return (
+      <Box
+        justifyContent='flex-start'
+        alignItems='center'
+        sx={{ backgroundColor: index % 2 ? theme.palette.container.main : 'none', p: 2, margin: 'auto', ...style }}
```

Comment:
Now that we've got a scroll bar to the right and have also changed how the container for each list item looks, I'd be quite keen to reduce/remove the horizontal padding we have on this list and the tabs above it.